resources/vars_dark.scss

```diff
@@ -0,0 +1,25 @@
+$color_primary0: #000;
+$color_primary5: #080808;   // light background
+$color_primary10: #111;     // background
+$color_primary25: #333;     // border
+$color_primary50: #808080;
+$color_primary75: #ccc;     // widget
+$color_primary90: #eee;
+$color_primary100: #fff;
+
+$color_link50: #06d;    // default (title)
+$color_link75: #38e;    // default
+$color_link100: #59f;   // hover
+$color_link200: #960;   // active
+
+$color_pageBg: #222;    // #222 because some elements should be darker than pageBg
+
+$highlight_blue: #2980B9;
+$announcement_red: #ae0000;
+
+$base_font_size: 14px;
```

Comment:
Could we have a `vars_common` for these?

Reply:
Done.